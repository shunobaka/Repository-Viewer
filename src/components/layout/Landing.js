import React from 'react';

const Landing = () => {
  return (
    <section className="landing">
      <div className="landing-inner">
        <h1>Welcome to Github-Repo-Viewer</h1>
        <h3>Search for a user in the navigation above to view repositories.</h3>
      </div>
    </section>
  );
};

export default Landing;
