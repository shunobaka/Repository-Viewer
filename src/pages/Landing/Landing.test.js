import React from 'react';
import renderer from 'react-test-renderer';
import Landing from './Landing';

describe('landing snapshots', () => {
  it('landing page renders correctly', () => {
    const tree = renderer.create(<Landing />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
