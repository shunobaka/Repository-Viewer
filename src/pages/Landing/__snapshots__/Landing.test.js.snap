// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`landing snapshots landing page renders correctly 1`] = `
<section
  className="landing"
>
  <div
    className="landing-inner"
  >
    <h1>
      Welcome to Github-Repo-Viewer
    </h1>
    <h3>
      Search for a user in the navigation above to view repositories.
    </h3>
  </div>
</section>
`;
